mobile.md

```diff
@@ -41,10 +41,66 @@ Below are the toolbar items currently supported, with any plugins/configuration
 | styleselect | Apply a custom style | | [style_formats]({{ site.baseurl }}/configure/content-formatting/#style_formats) |
 | removeformat | Removes any inline formatting |
 
+## Configuring the Plugins
+
+The plugins supported by mobile are currently limited to the [autosave]({{site.baseurl}}/plugins/autosave/), [autolink]({{site.baseurl}}/plugins/autolink/), and [lists]({{site.baseurl}}/plugins/lists/) plugins. This is by design. TinyMCE Mobile is in the early release cycle and does not include all the plugins available in the web version. The developer community is encouraged to provide suggestions about which plugins should be mobile optimized. Please complete this [survey](https://docs.google.com/forms/d/e/1FAIpQLSdWamU5HsZtv-SPqGRyu6Ql1zLqlrCQFP1vSrzx1oHikMFvlw/viewform) to provide feedback.
```

Comment:
"Desktop version" rather than "web version" maybe? As mobile still runs on the web.

Reply:
Fixed.